common/to.etc.alg/src/main/java/to/etc/net/HttpSsl.java

```diff
@@ -0,0 +1,141 @@
+package to.etc.net;
+
+import org.eclipse.jdt.annotation.Nullable;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.net.http.HttpClient;
+import java.net.http.HttpClient.Redirect;
+import java.net.http.HttpClient.Version;
+import java.security.KeyStore;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.Executor;
+import java.util.concurrent.ExecutorService;
+
+public class HttpSsl {
```

Comment:
some parts of this code exist also as JdkHttpClient. But if we do not use GenericHttpRequest abstraction, then we do not need these other parts... So, I placed here parts that could be used for simplified use of ssl client.